### PR TITLE
Updated certification page for front end certificate

### LINKED ISSUE #8261: Update Certifications Pages
#### Challenge Name
[Claim Your Front End Development Certificate](https://www.freecodecamp.com/challenges/claim-your-front-end-development-certificate)

#### Issue Description
At the URL: 
`https://www.freecodecamp.com/challenges/claim-your-front-end-development-certificate` in step 2 

there is supposed to have a link about Plagiarism, probably this one: 
`https://en.wikipedia.org/wiki/Plagiarism`

instead there is an empty anchor and doesn't open anything:
`https://www.freecodecamp.com/challenges/claim-your-front-end-development-certificate#`

#### Browser Information
Google Chrome 

* Browser Name, Version: 49.0.2623.112 m
* Operating System: Windows 10
* Mobile, Desktop, or Tablet: Desktop

### PR DESCRIPTION
**Pull request by NJFoster**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of FreeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/git-rebase#squashing-multiple-commits-into-one) them into one commit).
- [x] All new and existing tests pass the command `npm run test-challenges`. Use `git commit --amend` to amend any fixes.
#### Type of Change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Add new translation (feature adding new translations)
#### Checklist:

- [x] Tested changes locally.
- [x] Closes currently open issue (`Closes #XXXX`): Closes #8261
#### Description

Changed button text from 'Open link in new tab(this will open the next step)' to 'Confirm' and updated image backgrounds to green instead of the old brown color.

### CODE REVIEW

File: seed/challenges/01-front-end-development-certification/front-end-development-certificate.json [hunk 1]
```diff
@@ -14,19 +14,19 @@
...
         [
-          "//i.imgur.com/HArFfMN.jpg",
+          "//i.imgur.com/lPslPWR.jpg",
           "The definition of plagiarism: Plagiarism (noun) - copying someone else’s work and presenting it as your own without crediting them",
           "By clicking below, you pledge that all of your submitted code A) is code you or your pair personally wrote, or B) comes from open source libraries like jQuery, or C) has been clearly attributed to its original authors. You also give us permission to audit your challenge solutions and revoke your certificate if we discover evidence of plagiarism.",
           "#"
         ],
         [
-          "//i.imgur.com/14F2Van.jpg",
+          "//i.imgur.com/pl3OrQM.jpg
```
Comment (theflametrooper) [inline seed/challenges/01-front-end-development-certification/front-end-development-certificate.json]: ~~"//i.imgur.com/pl3OrQM.jpg~~ "//i.imgur.com/pl3OrQM.jpg",

**Comment by raisedadead** [comment]
@NJFoster Can you update this, you have left a `,` as pointed in the review comment above. Thanks!

**Comment by NJFoster** [comment]
@raisedadead Okay sorry, have been swamped with finals work.

File: server/views/challenges/showStep.jade [hunk 2]
```diff
@@ -14,7 +14,7 @@ block content
                     .spacer
                     .challenge-button-block
                         if step[3]
-                            a.btn.btn-block.btn-primary.challenge-step-btn-action.btn-lg(id='#{index}' href='#{step[3]}' target='_blank') Open link in new tab (this unlocks the next step)
+                            a.btn.btn-block.btn-primary.challenge-step-btn-action.btn-lg(id='#{index}' href='#{step[3]}' target='_blank') Confirm
```
Comment (raisedadead) [inline server/views/challenges/showStep.jade]: As mentioned by @BerkeleyTrue in his [comment](https://github.com/FreeCodeCamp/FreeCodeCamp/issues/8261#issuecomment-214136164), this has to be a conditional check.

The current change will affect all getting started challenges.

Comment (NJFoster) [inline server/views/challenges/showStep.jade]: Oh I am sorry I must have missed that I apologize. I will work on fixing it.

**Comment by theflametrooper** [comment]
@NJFoster You can remove the `Confirm` part of this PR. It has been fixed in #8606

**Comment by raisedadead** [comment]
Closing as this would require a rebase. 
Please have a look at 
https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/FreeCodeCamp-Fork-Local

Make changes as mentioned, and send a fresh PR.

**Pull request closed without merging**